Source/Client/Effects/FireEffect.cs

```diff
@@ -56,13 +57,17 @@ public FireEffect(Actor actor)
 			lightcolor = General.ARGB(1f, 1f, 0.9f, 0.5f);
 			light = new DynamicLight(actor.Position, 15f, 0, 3);
 
-			// Create sound
-			sound = DirectSound.GetSound("playerfire.wav", true);
-			sound.Position = this.actor.Position;
-			sound.Play(0f, true);
+            // Create sound
+            //sound = DirectSound.GetSound("playerfire.wav", true);
+            //sound.Position = this.actor.Position;
+            //sound.Play(0f, true);
```

Comment:
This PR has a fair amount of commented-out code. Please remove it if it is not needed, or mark with `// TODO: <what do you plan to do with it later>` if it _is_ still needed.

I won't mark every such section in this review, please take a look on your own.

This is one of the criteria to get this merged: no random unfinished code, and if it _is _ unfinished, then it should be marked as such, so that it's possible to open a new issue about it.